typescript/infra/src/utils/utils.ts

```diff
@@ -260,3 +265,11 @@ export function diagonalize<T>(array: Array<Array<T>>): Array<T> {
   }
   return diagonalized;
 }
+
+export function mustGetChainNativeTokenDecimals(chain: ChainName): number {
```

Comment:
Will we need this often? We could add something to `ChainMetadataManager` for it if you like

Reply:
It's only in one place so I'm ok keeping it out, if you want to bring it in though lmk